src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeTests.cs

```diff
@@ -167,9 +167,6 @@ public static void SameBinding_SecurityModeTransport_ClientCredentialTypeCertifi
     }
 
     [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Client_Certificate_Installed))]
```

Comment:
Already done. The ConditionalFact's were added after I had already verified the fix.